tests/e2escenarios/e2e_test.go

```diff
@@ -27,6 +27,14 @@ var _ = Describe("E2E Test", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
+	waitRemoteApp := func(urlInContainer, assertString string) {
+		cmp := helper.NewComponent(componentName, "app", "Dev", commonVar.Project, commonVar.CliRunner)
+		Eventually(func() string {
+			stdout, _ := cmp.Exec("runtime", nil, "curl", urlInContainer)
+			return stdout
+		}, 120*time.Second, 15*time.Second).Should(Equal(assertString))
+	}
```

Comment:
You could place this function in a place where it can be reused by other (integration) tests

Reply:
Good point - see [`e11c63f` (#6543)](https://github.com/redhat-developer/odo/pull/6543/commits/e11c63fb60d0b05c86e242e0bb3f31b362e08094)